clippy.toml

```diff
@@ -0,0 +1,502 @@
+#integer_division
+#let_underscore_must_use
+#lossy_float_literal
+#map_err_ignore
+#mem_forget
+#missing_docs_in_private_items
+#missing_enforced_import_renames
+#missing_inline_in_public_items
+#mod_module_files
+#modulo_arithmetic
+#multiple_inherent_impl
+#non_ascii_literal
+#panic
+#panic_in_result_fn
+#pattern_type_mismatch
+#print_stderr
+#print_stdout
+#rc_buffer
+#rc_mutex
+#rest_pat_in_fully_bound_structs
+#same_name_method
+#self_named_module_files
+#separated_literal_suffix
+#shadow_reuse
+#shadow_same
+#shadow_unrelated
+#str_to_string
+#string_add
+#string_slice
+#string_to_string
+#todo
+#undocumented_unsafe_blocks
+#unimplemented
+#unnecessary_self_imports
+#unneeded_field_pattern
+#unreachable
+#unseparated_literal_suffix
+#unwrap_in_result
+#unwrap_used
+#use_debug
+#verbose_file_reads
+#wildcard_enum_match_arm
+
+# === style ===
+
+#assertions_on_constants
+#assign_op_pattern
+#blacklisted_name
+#blocks_in_if_conditions
+#bool_assert_comparison
+#borrow_interior_mutable_const
+#builtin_type_shadow
+#bytes_nth
+#chars_last_cmp
+#chars_next_cmp
+#cmp_null
+#collapsible_else_if
+#collapsible_if
+#collapsible_match
+#comparison_chain
+#comparison_to_empty
+#declare_interior_mutable_const
+#double_must_use
+#double_neg
+#duplicate_underscore_argument
+#enum_variant_names
+#excessive_precision
+#field_reassign_with_default
+#fn_to_numeric_cast
+#fn_to_numeric_cast_with_truncation
+#for_kv_map
+#from_over_into
+#from_str_radix_10
+#inconsistent_digit_grouping
+#infallible_destructuring_match
+#inherent_to_string
+#into_iter_on_ref
+#iter_cloned_collect
+#iter_next_slice
+#iter_nth_zero
+#iter_skip_next
+#just_underscores_and_digits
+#len_without_is_empty
+#len_zero
+#let_and_return
+#main_recursion
+#manual_async_fn
+#manual_map
+#manual_non_exhaustive
+#manual_range_contains
+#manual_saturating_arithmetic
+#map_clone
+#map_collect_result_unit
+#match_like_matches_macro
+#match_overlapping_arm
+#match_ref_pats
+#match_result_ok
+#mem_replace_option_with_none
+#mem_replace_with_default
+#missing_safety_doc
+#mixed_case_hex_literals
+#module_inception
+#must_use_unit
+#mut_mutex_lock
+#needless_borrow
+#needless_doctest_main
+#needless_late_init
+#needless_range_loop
+#needless_return
+#neg_multiply
+#new_ret_no_self
+#new_without_default
+#ok_expect
+#op_ref
+#option_map_or_none
+#print_literal
+#print_with_newline
+#println_empty_string
+#ptr_arg
+#ptr_eq
+#question_mark
+#redundant_closure
+#redundant_field_names
+#redundant_pattern
+#redundant_pattern_matching
+#redundant_static_lifetimes
+#result_map_or_into_option
+#result_unit_err
+#same_item_push
+#self_named_constructors
+#should_implement_trait
+#single_char_add_str
+#single_component_path_imports
+#single_match
+#string_extend_chars
+#tabs_in_doc_comments
+#to_digit_is_some
+#toplevel_ref_arg
+#try_err
+#unnecessary_fold
+#unnecessary_lazy_evaluations
+#unnecessary_mut_passed
+#unsafe_removed_from_name
+#unused_unit
+#unusual_byte_groupings
+#unwrap_or_else_default
```

Comment:
Unwrap or else is lazy by its nature, is most cases not useful to make return in a deterministic way. I'd add just one more ## to mark this setting. =)